platform/viewer/cypress/support/commands.js

```diff
@@ -328,6 +328,21 @@ Cypress.Commands.add('percyCanvasSnapshot', (name, options = {}) => {
   });
 });
 
+Cypress.Commands.add('setLayout', (columns = 1, rows = 1) => {
```

Comment:
👍

Reply:
cc @mirnasilva this custom command can be helpful 😄